package.json

```diff
@@ -37,6 +37,11 @@
     "rollup": "^2.30.0"
   },
   "dependencies": {
-    "express": "^4.16.4"
+    "alert": "^5.0.5",
+    "body-parser": "^1.19.0",
+    "express": "^4.16.4",
+    "express-validator": "^6.6.1",
+    "mysql": "^2.18.1",
+    "nodemon": "^2.0.6"
```

Comment:
As `nodemon` is not needed for deployment it shouldn't be listed in the dependencies, it could be made into a dev dependency, but currently our `package.json` is not setup to use it. Should probably remove it from here (you can still install it locally for your own development use).

```suggestion
```